operations/multi-tenancy/ExampleWorkflow.md

```diff
@@ -0,0 +1,806 @@
+# Example Workflow
+
+* [Overview](#overview)
+* [Configuration](#configuration)
+    * [Tenant Configuration](#tenant-configuration)
+        * [Tenant Names](#tenant-names)
+        * [Tenant API version](#tenant-api-version)
+    * [`Slurmcluster` Configuration](#slurmcluster-configuration)
+        * [`Slurmcluster` Names](#slurmcluster-names)
+        * [`Slurmcluster` IP Addresses](#slurmcluster-ip-addresses)
+        * [`Slurmcluster` API version](#slurmcluster-api-version)
+        * [`Slurmcluster` Configurable Values](#slurmcluster-configurable-values)
+        * [`Slurmcluster` Version Numbers](#slurmcluster-version-numbers)
+    * [`Slurmcluster` Slurm configuration](#slurmcluster-slurm-configuration)
+        * [`Slurmcluster` Slingshot VNI Allocation](#slurmcluster-slingshot-vni-allocation)
+        * [`Slurmcluster` Partitions and Nodes](#slurmcluster-partitions-and-nodes)
+        * [`Slurmcluster` Secrets (for `nonroot` users)](#slurmcluster-secrets-for-nonroot-users)
+    * [USS Configuration](#uss-configuration)
+* [Step-by-Step Guide](#step-by-step-guide)
+    * [Create tenant configuration file](#create-and-apply-tenant-configuration-file)
+    * [Create `Slurmcluster` configuration file](#create-and-apply-slurmcluster-configuration-file)
+    * [Edit Slurm configuration file](#edit-and-apply-slurm-configuration-file)
+    * [Edit Slurm configuration file `sssd.conf`](#edit-slurm-configuration-file-sssdconf)
+    * [Configure USS group variables](#configure-uss-group-variables)
+    * [Create BOS session template](#create-bos-session-template)
+    * [Create CFS configuration](#create-cfs-configuration)
+    * [Boot and Run](#boot-and-run)
+* [Status and Troubleshooting](#status-and-troubleshooting)
+    * [Tenant command examples](#tenant-command-examples)
+    * [`Slurmcluster` command examples](#slurmcluster-command-examples)
+    * [HSM command examples](#hsm-command-examples)
+    * [HNS command example](#hns-command-example)
+* [Appendices](#appendices)
+    * [Appendix A - `Noname` tenant](#appendix-a---noname-tenant)
+    * [Appendix B - `Noname Slurmcluster`](#appendix-b---noname-slurmcluster)
+    * [Appendix C - Slurm configuration](#appendix-c---slurm-configuration)
+    * [Appendix D - USS group variables](#appendix-d---uss-group-variables)
+
+## Overview
+
+A tenant is a collection of nodes that is dedicated to one particular set of users on an HPE Cray EX computer system running CSM.
+This guide is intended to provide a comprehensive set of instructions for a system administrator to configure, deploy, and run applications on, one or two tenants.
+
+In this document we provide examples for a hypothetical system called `Noname`, which has two tenants, and each tenant has a `Slurmcluster`.
```

Comment:
Is noname an internal system name? We might want to change this name if it is since this is a public repository.

Reply:
I'm not sure why this name was chosen since I'm going off of the draft someone else gave me.  I've changed this to "Development" to be safe.